observationtree/src/main/scala/explore/observationtree/TargetObsList.scala

```diff
@@ -151,7 +152,7 @@ object TargetObsList {
       val observations = props.observations.get
       val obsByTarget  = observations.groupBy(_.target)
 
-      <.div(^.width := "270px", ^.marginTop := "30px")(
+      <.div(GPPStyles.ObsTree, ^.marginTop := "30px")(
```

Comment:
Maybe we should put the top margin in the css

Reply:
I wasn't sure about leaving the margin. I just put it there so that the reusability counters wouldn't block visuals in development. But we might leave it, it didn't look bad without the counters.